lib/platform/github/index.spec.ts

```diff
@@ -130,6 +130,16 @@ describe(getName(), () => {
       ).toMatchSnapshot();
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('should add initialized platform with predefined generic host rule for github api', async () => {
```

Comment:
```suggestion

    it('should add initialized platform with predefined generic host rule for github api', async () => {
```

Readabillity, Add newline before each new test. Ignore existing